music_assistant/common/models/media_items.py

```diff
@@ -269,20 +274,16 @@ class _MediaItemBase(DataClassDictMixin):
 
     item_id: str
     provider: str  # provider instance id or provider domain
-    name: str
+    name: str = "[Unknown]"
     version: str = ""
     # sort_name and uri are auto generated, do not override unless really needed
     sort_name: str | None = None
     uri: str | None = None
     external_ids: set[tuple[ExternalID, str]] = field(default_factory=set)
     media_type: MediaType = MediaType.UNKNOWN
 
-    def __post_init__(self):
+    def __post_init__(self) -> None:
         """Call after init."""
-        if self.name is None:
```

Comment:
I had to lift this up or make name `str | None`. If the problem was that plex is explicitly passing None, then I think we have to fix that in the provider itself but I don't know if that was the problem or not.

Reply:
Yeah, this is an annoying one - one of the providers actually has a few edge cases where the name was None, causing a flow of edge case messy hell. Maybe I shouldnt have put this workaround here but explicitly handle it in the provider where it happened. Now, I just need to remember which provider had this issue :-)

Reply:
I think it was Plex. I can add a similar workaround there.

Reply:
I think that is better - the models should just assume that the provider doesn't do stupid things.

Reply:
4f80bdfe8550eae805d8927bbd4751c213b645ae